vendor/lowrisc_ip.vendor.hjson

```diff
@@ -0,0 +1,23 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+    name: "lowrisc_ip",
+    target_dir: "lowrisc_ip",
+    patch_dir: "patches/lowrisc_ip",
+
+    upstream: {
+        url: "https://github.com/lowRISC/opentitan"
+        rev: "master"
+    }
+
+    mapping: [
+        {from: "util/tlgen.py", to: "util/tlgen.py"},
+        {from: "util/tlgen",    to: "util/tlgen" patch_dir: "tlgen"},
+        {from: "util/reggen",   to: "util/reggen"},
```

Comment:
There are a few mappings that look like they do nothing. Do they do something?

Reply:
If you don't specify a path in the mapping, it does not get imported so that is why some of the `from` and `to` fields are the same.